p/themes/BlueLagoon/BlueLagoon.css

```diff
@@ -566,6 +567,11 @@ a.btn {
 	color: #0090FF
 }
 
+/*=== Scrollbar */
+.ss-scroll, #sidebar:hover::-webkit-scrollbar-thumb {
```

Comment:
Forgotten split here, like for the other themes?

Reply:
Yep, certainly did!